keras_cv/layers/preprocessing/jittered_resize_test.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv import bounding_box
+from keras_cv import layers
+
+
+class JitteredResizeTest(tf.test.TestCase, parameterized.TestCase):
```

Comment:
Let's also add this new layer to MixedPrecisionTest

Reply:
not seeing that test anywhere - am I missing something?

Reply:
@ianstenbit Is this still required?

Reply:
This got added in [this commit](https://github.com/keras-team/keras-cv/pull/1053/commits/bdc699fa0b6f0d1849670aa05cf08643ce150c58) in this PR, so good on this end.

In terms of the test being required, yeah I think it's good to keep the mixed precision tests covering as many layers as possible